kfserving/kfserving-crds/overlays/application/application.yaml

```diff
@@ -3,29 +3,29 @@ kind: Application
 metadata:
   name: kfserving-crds
 spec:
-  selector:
-    matchLabels:
-      app.kubernetes.io/name: kfserving-crds
-      app.kubernetes.io/instance: kfserving-crds-v1.0.0
-      app.kubernetes.io/managed-by: kfctl
-      app.kubernetes.io/component: kfserving-crds
-      app.kubernetes.io/part-of: kubeflow
-      app.kubernetes.io/version: v1.0.0
+  addOwnerRef: true
   componentKinds:
   - group: core
     kind: ConfigMap
   - group: apps
     kind: Deployment
   descriptor:
-    type: kfserving-crds
-    version: v1beta1
-    description: ""
-    maintainers: []
-    owners: []
+    description: ''
     keywords:
-     - kfserving-crds
-     - kubeflow
+    - kfserving-crds
+    - kubeflow
     links:
     - description: About
-      url: ""
-  addOwnerRef: true
+      url: ''
+    maintainers: []
+    owners: []
+    type: kfserving-crds
+    version: v1beta1
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: kfserving-crds
+      app.kubernetes.io/instance: kfserving-crds-0.2.2
+      app.kubernetes.io/managed-by: kfctl
+      app.kubernetes.io/name: kfserving-crds
+      app.kubernetes.io/part-of: kubeflow
+      app.kubernetes.io/version: 0.2.2
```

Comment:
believe this is the version for application, i.e. kfserving

Reply:
That is correct. The application in this case is KFServing so the version should be KFServing version not Kubeflow version.

Ideally "Kubeflow" is its own application which should be the parent of many applications.
There might be some ongoing work to clean up /fix labels to make that work.